Jenkinsfile

```diff
@@ -59,10 +69,36 @@ pipeline {
 
     stage('Deploy (master only)') {
       when {
-        branch 'master'
+        allOf {
+          branch 'master' 
+          expression {return currentBuild.currentResult == 'SUCCESS' && params.publish}
+        }
       }
       steps { 
         script {
+          docker.build('node').inside {
+            sh "git config --global user.name '${params.gitUserName}'"
+            sh "git config --global user.email '${params.gitUserMail}'"
+            sh "echo //npm-registry.ivyteam.io/repository/private/:_authToken=${env.NPM_TOKEN} > .npmrc"
+            sh "yarn lerna version ${params.nextVersion} --yes && yarn publish:package"
+            sh "rm .npmrc"
+            
+            dir ('integration/vscode/webview') {
+              sh "yarn upgrade @ivyteam/process-editor@${params.nextVersion}"
+            }
+            dir ('integration/vscode') {
+              sh "yarn"
+              sh "git commit --all --amend --no-edit"
+              sh "git tag -f v${params.nextVersion}"
+            }
+            withEnv(['GIT_SSH_COMMAND=ssh -o StrictHostKeyChecking=no']) {
+              sshagent(credentials: ['github-axonivy']) {
+                sh "git remote set-url origin git@github.com:axonivy/glsp-editor-client.git"
+                sh "git push -u origin ${env.BRANCH_NAME}"
```

Comment:
Better push to another branch, not master.

Reply:
![image](https://user-images.githubusercontent.com/42733123/123791684-07cbc200-d8e0-11eb-9ac7-1e3314753f70.png)